app/helpers/organization_helper.rb

```diff
@@ -0,0 +1,6 @@
+module OrganizationHelper
+  def display_logo_or_name organization = nil
+    organization ||= current_organization
+    organization.logo.present? ? image_tag(organization.logo, alt: "#{organization.name} logo", class: "organization-logo") : organization.name
```

Comment:
With the diaperbase logo in the upper-left already, it might be kinda weird to have it also appear in the content area -- this will default to the org name if they haven't provided a logo tho